app/src/main/AndroidManifest.xml

```diff
@@ -16,6 +16,7 @@
             android:name=".MainActivity"
             android:exported="true"
             android:label="@string/app_name"
+            android:screenOrientation="landscape"
```

Comment:
Just curious was it intended to restrict the app to landscape mode? The README mentioned the apps works in portrait and landscape modes but this directive locks to landscape.

Reply:
Ah..missed removing this before PR. Was testing something for landscape and hence had added that. Will remove this. Thank you.